client/utils/audioMachine.js

```diff
@@ -0,0 +1,30 @@
+module.exports = class AudioMachine {
+  constructor () {
+    this.playlist = {}
+  }
+
+  addSound (name, src) {
+    const newSound = {}
+    const soundElement = document.createElement('audio')
```

Comment:
One thing to note - what if this is a node.js app consuming the API library?

Reply:
Let me do some digging

Reply:
or... find something like https://www.npmjs.com/package/play - which is pretty old but still has some 270 weekly downloads